package.json

```diff
@@ -30,7 +30,7 @@
     "husky": "1.3.1",
     "istanbul": "1.1.0-alpha.1",
     "jsdom": "^15.1.1",
-    "lerna": "3.22.1",
+    "lerna": "5.1.8",
```

Comment:
I haven't see a max buffer issue since I did this update.  Probably should also update the global version and the docs.